docs/README.md

```diff
@@ -1,16 +1,31 @@
-
-
-This directory is good for storing docs.
-
-
-
-## Moved from GitHub Wiki
-
-In November 2018 we moved some docs from the GitHub wiki over to the
-repository.
-
-Thanks for moving stuff elsewhere that's better off elsewhere.
-
-Even more thanks for removing stuff that that is clearly outdated and
-irrelevant.
-
+---
+title: Trustroots development documentation
+---
```

Comment:
Hope this works; just not quite sure how to test without merging. :-/

https://jekyllrb.com/docs/front-matter/

Reply:
Totally 👍